apps/lend/src/constants.ts

```diff
@@ -52,7 +52,7 @@ export enum TITLE {
   tokenSupply = 'tokenSupply',
   totalCollateralValue = 'totalCollateralValue',
   totalDebt = 'totalDebt',
```

Comment:
sorry for this one, pushed to the wrong branch. I just reverted because I cannot force-push.